src/screens/Dashboard/Dashboard.js

```diff
@@ -286,31 +313,31 @@ export default function HrDashboard(props) {
   };
   const [approveRequestModal, setApproveRequestModal] = useState({
     show: false,
-    data: null,
+    data: null
   });
 
   const [historyModal, setHistoryModal] = useState({
     show: false,
-    data: null,
+    data: null
   });
 
   const [regularizationRequest, setRegularizationRequest] = useState([]);
   const [ticketID, setTicketID] = useState();
   const [isLoading, setIsLoading] = useState(false);
   const [notificationId, setNotificationId] = useState();
 
-  const handleRegularizationRequest = (cuurentData) => {
-    
-    
+  const handleRegularizationRequest = async (currentData) => {
+    const id =
+      currentData?.ticketID === undefined ? currentData : currentData.ticketID;
```

Comment:
1.You are checking for undefined,
currentData is a object : {ticketId : "",notificationID : 123};
so ticketId can also be emtystring if not availble ,
Rather check for falsy value.

Reply:
resolved